src/services/azureFunctionsService.ts

```diff
@@ -77,9 +78,10 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 		const newFilePromise = azureFunctionUtils.waitForNewFunctionFile(projectFile);
 
 		// get function name from user
+		let uniqueFunctionName = await getUniqueFsPath(path.dirname(projectFile), table);
 		const functionName = await vscode.window.showInputBox({
 			title: constants.functionNameTitle,
-			value: table,
+			value: uniqueFunctionName,
```

Comment:
While you're here - it would be good to add a validation check for the name to be non-empty